### PR TITLE
feat(cc): add data source to get a list of CC central network attachments

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
add data source to get a list of CC central network attachments
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add data source to get a list of CC central network attachments
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
 make testacc TEST="./huaweicloud/services/acceptance/cc" TESTARGS="-run TestAccDataSourceCcCentralNetworkAttachments_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cc -v -run TestAccDataSourceCcCentralNetworkAttachments_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceCcCentralNetworkAttachments_basic
=== PAUSE TestAccDataSourceCcCentralNetworkAttachments_basic
=== CONT  TestAccDataSourceCcCentralNetworkAttachments_basic
--- PASS: TestAccDataSourceCcCentralNetworkAttachments_basic (342.62s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cc        342.681s

```

### CODE REVIEW

File: huaweicloud/services/acceptance/cc/data_source_huaweicloud_cc_central_network_attachments_test.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+data "huaweicloud_cc_central_network_attachments" "filter_by_id" {
+  central_network_id = huaweicloud_cc_central_network.test.id
+  attachment_id      = local.id
+
+  depends_on = [
+    huaweicloud_cc_central_network_attachment.test,
+  ]
+}
+
+output "is_id_useful" {
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/cc/data_source_huaweicloud_cc_central_network_attachments_test.go]:
```suggestion
output "is_id_filter_useful" {
```
and fix other places

Comment (profoundwu) [reply huaweicloud/services/acceptance/cc/data_source_huaweicloud_cc_central_network_attachments_test.go]: fixed

**Pull request merged**